member/templates/member/events/booking_confirm.html

```diff
@@ -55,7 +56,7 @@ <h4 class='header-font' style='text-align:center; color:black;'>Confirm Booking<
                 {% if forloop.counter0|divisibleby:loop_count %}
                   <th class="rotate"><div><span>
                     {{ time.hour }}:{{ time.minutes }}
-                  </div></span></th>
```

Comment:
@jsayles Do you want to remove this from the PR ?

Reply:
No this is OK. Thank you.